docs/files/index.md

```diff
@@ -1,6 +1,11 @@
 ---
 title: Files
-intro: Configuration files are crucial for consistent and accurate resource deployments, containing necessary information and settings.
+intro: |
+    Configuration files are used to store configuration settings and environment variables that are used by application and infrastructure resources.
+
+    <br/><br/>Example: a `.env` file (also known as "dotenv") containing environment variables for an `Application`).
+
+    <br/><br/>Using Devopness you can easily manage and organize multiple configuration files on your environment resources defining custom variables to be set during a deploy, without the need of hard-coding them directly into your application code and without the need of SSH into your server every time an environment variable needs to be updated.
```

Comment:
```suggestion
    <br/><br/>
    
    Example: a `.env` file (also known as "dotenv") containing environment variables for an `Application`).

    <br/><br/>
    
    Using Devopness you can easily manage and organize multiple configuration files on your environment resources defining custom variables to be set during a deploy, without the need of hard-coding them directly into your application code and without the need of SSH into your server every time an environment variable needs to be updated.
```

Since `<br/>` is not part of Markdown syntax, it would be beneficial to place them on separate lines, for it makes the `line break` intention clearer for those who may not be familiar with HTML.

What do you think?

Reply:
> Since <br/> is not part of Markdown syntax

@Diegiwg our website currently renders markdown content following the [CommonMark spec](http://spec.commonmark.org/), and HTML rendering is defined there.

> it would be beneficial to place them on separate lines

I agree that the use of HTML may not be the best solution to adding line breaks in a markdown file and the positioning of such tags may be open to discussion, but I don't think that's enough of a reason to hold this PR from being merged.

For now, will close the issue and merge the PR.